capa/features/address.py

```diff
@@ -27,6 +27,9 @@ def __new__(cls, v):
         assert v >= 0
         return int.__new__(cls, v)
 
+    def __str__(self):
+        return str(self)
```

Comment:
what does this do? im surprised it doesn't work already?

Reply:
mypy was complaining about this in CI: https://github.com/mandiant/capa/runs/6410100333?check_suite_focus=true#step:8:8
```
...
capa/features/extractors/ida/insn.py:446: error: Cannot instantiate abstract class "AbsoluteVirtualAddress" with abstract attribute "__str__"
capa/features/extractors/ida/function.py:23: error: Cannot instantiate abstract class "AbsoluteVirtualAddress" with abstract attribute "__str__"
capa/features/extractors/common.py:24: error: Cannot instantiate abstract class "FileOffsetAddress" with abstract attribute "__str__"
capa/features/extractors/common.py:27: error: Cannot instantiate abstract class "FileOffsetAddress" with abstract attribute "__str__"
capa/features/extractors/viv/file.py:25: error: Cannot instantiate abstract class "FileOffsetAddress" with abstract attribute "__str__"
...
```